tools/autopep8.sh

```diff
@@ -59,6 +59,32 @@ function main() {
   fi
 
   DIR=${@:-.}
+
+  # Only run autopep8 on tracked files. This saves time and possibly avoids
+  # formatting files the user doesn't want formatted.
+  tmp_dir=/tmp/git_files_$$
```

Comment:
this should probably use `mktemp -d` (eg `tmp_dir=$(mktemp -d)`)

Reply:
Yeah, that's better.